learning_journal/learning_journal/__init__.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from pyramid.config import Configurator
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application."""
+    settings["sqlalchemy.url"] = os.environ["DATABASE_URL"]
```

Comment:
you are reading your db url from the environment here, so you can safely remove the url from your `.ini` file above.